doc/policy/README.md

```diff
@@ -0,0 +1,12 @@
+# Mempool Policy
+
+**Mempool Policy** (aka Transaction Relay Policy) is the node's set of validation rules, in addition
+to consensus, enforced for unconfirmed transactions before submitting them to the mempool.  These
+local rules include restrictions on the transaction itself and the transaction in relation to the
+node's mempool contents. They are *not* applied to transactions in blocks.
```

Comment:
I would also say "and chain tip" as your chain tip is changing the outcome of some RBF rules, such as rule 2 on unconfirmed inputs.

Reply:
Added